### DISCUSSION TITLE
Lists of figures and tables

### DISCUSSION THREAD

**Discussion by canhaoaugusto** [joaomlourenco/novathesis#327] [Q&A] [answered]
Hi everyone!

I'm having a problem: I need to insert a blank page between list of figures and list of tables, and I really don't know how to do it...

Thank you :-)

**Comment by joaomlourenco** [comment]
You are asking for something that is the "solution" to some problem you have/feel.  Which problem is that? Why do you need to add a blank page between those two lists?

**Reply by canhaoaugusto** [reply]
To my MSc thesis, I need to start TOC, list of figures and list of tables in odd pages. Since my TOC has 2 pages, I already have list of figures starting in an odd page, but my list of tables start in an even page (the one next to list of figures, that has only 1 page)

**Reply by joaomlourenco** [reply]
So, what you need is not to insert a blank page between the LoF and the LoT.  What you need is a "paper" version of the PDF, where all the relevant material (List of…, Abstracts, Chapters, etc) start in a odd numbered page.  :) 
See the answer below…

**Reply by joaomlourenco** [reply]
If the reply below satisfies your needs, please press the "Mark as answer" button in the bottom left of the message box.

**Reply by canhaoaugusto** [reply]
It's perfect! Thank you so much!!

**Comment by joaomlourenco** [comment, marked as answer]
To have all the _relevant_ material (List of…, Abstracts, Chapters, etc) starting in an odd numbered page, add
```latex
\openright
```
somewhere in the file `Config/5_packages.tex".
